docs/index.rst

```diff
@@ -26,7 +26,7 @@ Kendraio App
 
 .. toctree::
    :maxdepth: 2
-   :caption: Workflow Builder
+   :caption: Flow Builder (was Workflow Builder)
```

Comment:
I feel weird in giving an English comment given that you are all mother tongue and I am not :D
Anyway usually in these situations people write: "formerly Workflow Builder" and it does sounds better to me. 
Don't know if make a big difference in term of English language/communication tho.

Reply:
I agree, it's only a small thing but "formerly" sounds more professional.